packages/cli/src/cmds/validator/options.ts

```diff
@@ -309,11 +304,16 @@ export const validatorOptions: CliCommandOptions<IValidatorCliArgs> = {
 
   disableKeystoresThreadPool: {
     hidden: true,
-    description:
-      "Disable thread pool and instead use main thread to decrypt keystores. This can speed up decryption in testing environments like Kurtosis",
+    description: "Disable thread pool and instead use main thread to decrypt keystores. This can speed up decryption in testing environments like Kurtosis",
     type: "boolean",
   },
 
+  keystoresLockFilePath: {
+    hidden: true,
+    description: "To create .lock file for Keystores files being read in ConfigMap",
```

Comment:
this is not what we want, we need a flag to allow a user to specify a custom directory to create `.lock` files in instead of the default one which is always the same directory as `--importKeystores`